### PR TITLE
[SOLITAIRE] French translation update

### PR DESCRIPTION
**Pull request by KyleKatarn31**
## Purpose

- Current French translation is incomplete / incorrect

solitaire

## Proposed changes

- Complete / correct french translation

Sans titre

### CODE REVIEW

**Comment by KyleKatarn31** [comment]
@reactos/lang-french @sdever @HBelusca : for review / approval / merge. Thanks !

**Comment by binarymaster** [comment]
Cc @reactos/lang-french

File: base/applications/games/solitaire/lang/fr-FR.rc [hunk 1]
```diff
@@ -16,15 +16,15 @@ CAPTION "Options"
 FONT 8, "MS Shell Dlg"
 BEGIN
     GROUPBOX "Cartes", -1, 7, 7, 90, 40
-    AUTORADIOBUTTON "En tirer une", IDC_OPT_DRAWONE, 14, 19, 70, 10, WS_GROUP | WS_TABSTOP
-    AUTORADIOBUTTON "En tirer trois", IDC_OPT_DRAWTHREE, 14, 32, 70, 10
-    GROUPBOX "Scoring", -1, 100, 7, 75, 53
+    AUTORADIOBUTTON "Tirer une carte", IDC_OPT_DRAWONE, 14, 19, 70, 10, WS_GROUP | WS_TABSTOP
+    AUTORADIOBUTTON "Tirer trois cartes", IDC_OPT_DRAWTHREE, 14, 32, 70, 10
```
Comment (HBelusca) [inline base/applications/games/solitaire/lang/fr-FR.rc:20]: No idea what Windows does, but I personally tend to prefer the previous translations.

File: base/applications/games/solitaire/lang/fr-FR.rc [hunk 2]
```diff
@@ -60,7 +60,7 @@ BEGIN
     IDS_SOL_WIN "Félicitations ! Vous avez gagné !"
     IDS_SOL_DEAL "Redistribuer ?"
     IDS_SOL_SCORE "Score: %d"
```
Comment (HBelusca) [inline base/applications/games/solitaire/lang/fr-FR.rc:62]:
```suggestion
    IDS_SOL_SCORE "Score : %d"
```

File: base/applications/games/solitaire/lang/fr-FR.rc [hunk 3]
```diff
@@ -60,7 +60,7 @@ BEGIN
...
-    IDS_SOL_TIME "Time: %d"
+    IDS_SOL_TIME "Temps: %d"
```
Comment (HBelusca) [inline base/applications/games/solitaire/lang/fr-FR.rc:63]:
```suggestion
    IDS_SOL_TIME "Temps : %d"
```

File: base/applications/games/solitaire/lang/fr-FR.rc (hunk 1 above)
Comment (KyleKatarn31) [reply base/applications/games/solitaire/lang/fr-FR.rc]: As you prefer. I reverted.

**Comment by KyleKatarn31** [comment]
@HBelusca : All your comments are in. Ready to go ?

**Pull request merged**